### ISSUE TITLE
SetSavePoint not marking document as unmodified

### ISSUE THREAD

**Issue by theryan722** [jacobslusser/ScintillaNET#51]
The document still reports being modified even when you call **SetSavePoint**. I even tested to make sure the **SavePointReached** event was firing, which it was. Am I doing something wrong here?

**Comment by jacobslusser** [comment]
I don't see any problem.

```cs
scintilla.SetSavePoint();
Debug.WriteLine(scintilla.Modified);
```

The code above always displays `false` for me. The `Modified` property is also false for me when I display the value in the `SavePointReached` event handler.

**Comment by theryan722** [comment]
Weird it is always displaying true for me. I will look further into this,
thanks.

**Comment by theryan722** [comment]
Okay, so after further testing, I discovered that by checking through a button:

```c#
Interaction.MsgBox(Scintilla1.Modified)
```

Works, but doing so through a timer, does not. Maybe this is just me? The timer was not going to be in the final code, it was just to test things out. I had the timer set the text of a label, displaying whether the textbox was modified or not, and it would always display modified, but the msgbox worked. This is probably just on my end, but I'm just reporting what I observed.
